### PR TITLE
Update using-dev-build

### PR DESCRIPTION
**Pull request by nmattia**
This updates the `using-dev-build` tests:
* The test runner is ported to typescript for maintainability
* chromedriver is updated to 104
* Formatter is run

### CODE REVIEW

File: demos/using-dev-build/test.ts [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+const WEBAPP_PORT = 8087;
+const WEBAPP_URL = `http://localhost:${WEBAPP_PORT}`;
+
+type Arguments = { noRun: boolean };
+
+/** Parse CLI arguments
+ */
+function parseArguments(): Arguments {
+  if (!(process.argv.length == 2 || process.argv.length == 3)) {
```
Comment (peterpeterparker) [inline demos/using-dev-build/test.ts:24]: - `===` not `==` (on line 28 as well)
- fancy suggestion: `!([2, 3].includes(process.argv.length))`

File: demos/using-dev-build/test.ts [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    };
+  } catch (e) {
+    console.log(
+      `Could not read 'internet_identity' and 'webapp' local canister IDs from ${CANISTER_IDS_PATH}`
+    );
+    throw e;
+  }
+}
+
+function parseReplicaHost(): string {
```
Comment (peterpeterparker) [inline demos/using-dev-build/test.ts:60]: it actually returns `string | undefined` I think

**Review by peterpeterparker — APPROVED** [review]
Rubber stamp, two minor comments I read on the fly

File: demos/using-dev-build/test.ts (hunk 2 above)
Comment (nmattia) [reply demos/using-dev-build/test.ts:60]: damn you're right! typescript failed me

Comment (nmattia) [reply demos/using-dev-build/test.ts:60]: Updated

Comment (peterpeterparker) [reply demos/using-dev-build/test.ts:60]: 👍

**Pull request merged**